docs/streamtypes/tile-document/overview.md

```diff
@@ -1,25 +1,39 @@
-# Tile Document
+# TileDocument
 
-Tile Document is a StreamType that stores a JSON document, providing similar functionality as a NoSQL document store.
+TileDocument is a StreamType that stores a mutable JSON document, providing similar functionality as a NoSQL document store.
 
-## Use cases
+## **Usage**
 
-Tile Document streams are commonly used as a NoSQL database replacement for:
+See [TileDocument API](./api.md) for instructions on how to create and update TileDocument streams. You can query TileDocuments using Ceramic's standard [queries API](../../build/queries.md).
 
-- Identity metadata (i.e. profiles, social graphs, reputation scores, linked social accounts)
+## **Specification**
+
+See [TileDocument StreamType (CIP-8)](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-8/CIP-8.md) for the complete specification.
+
+### Storage
+
+TileDocuments are streams used for storing JSON documents. The TileDocument stream is structured as a single log of [commits](../../learn/glossary.md#commits), where each commit only contains the diff from the previous version. Optionally, TileDocuments may specify a JSON schema and all commits must adhere to the schema.
+
+### Consensus
+
+TileDocuments rely on [anchor commits](../../learn/glossary.md#anchor-commit) for providing immutable timestamps for the [genesis commit](../../learn/glossary.md#genesis-commit) and subsequent [signed commits](../../learn/glossary.md#signed-commit) in the stream. In the case of conflicting versions, the branch with the earliest recorded anchor commit will be respected as the canonical branch.
+
+### Authentication
+
+TileDocuments rely on [DIDs](../../learn/glossary.md#dids) for [authentication](../../learn/glossary.md#authentication). Only the DID(s) assigned as the [controller](../../learn/glossary.md#controllers) of the stream are allowed to perform writes. 
+
+## **Sample use cases**
+
+TileDocuments are commonly used for storing:
+
+- Schemas
+- Identity metadata and information
 - User-generated content (i.e. blog posts, social media, etc)
-- Indexes of other streams to form collections and user tables (i.e. IDX)
```

Comment:
I feel like the reference to IDX is nice to include? It does show off a pretty powerful use case for TileDocuments

Reply:
fixed